georiviere/river/forms.py

```diff
@@ -9,7 +11,19 @@
 
 class StreamForm(CommonForm):
     geom = SnappedLineStringField()
-    geomfields = ['geom', 'source_location']
+    geomfields = ['geom']
```

Comment:
you delete source_location ?

Reply:
I remove it in geomfields because it's already **not** in the fieldlayout.

My first fix was good but it was better to use directly fieldlayout